src/NzbDrone.Core/Parser/LanguageParser.cs

```diff
@@ -146,6 +148,23 @@ public static Language ParseSubtitleLanguage(string fileName)
 
             return Language.Unknown;
         }
+        
+        public static IEnumerable<string> ParseLanguageTags(string fileName)
+        {
+            try
+            {
+                var simpleFilename = Path.GetFileNameWithoutExtension(fileName);
+                var match = SubtitleLanguageRegex.Match(simpleFilename);
+                var languageTags = from Capture tag in match.Groups["tags"].Captures select tag.Value.ToLower();
+                return languageTags.Where(tag => !tag.Empty());
+            }
+            catch (Exception ex)
+            {
+                Logger.Debug(ex, "Failed parsing language tags from subtitle file: {0}", fileName);
+            }
+
+            return new string[] {};
```

Comment:
`IEnumerable.Empty` (not sure the exact syntax offhand)

Reply:
Fixed in 6f950a9.